ssl/ssl_cipher.cc

```diff
@@ -1193,6 +1192,25 @@ static bool ssl_cipher_process_rulestr(const char *rule_str,
   return true;
 }
 
+static const char *kKnownKeywordFilterRulesMappingToDefault[] = {
+  "ALL",
+  "DEFAULT",
+  "FIPS",
+  "HIGH",
+};
+
+static bool is_known_default_alias_keyword_filter_rule(const char *rule,
+  size_t *matched_rule_length) {
+
+  for (auto known_rule : kKnownKeywordFilterRulesMappingToDefault) {
+    if (strncmp(rule, known_rule, strlen(known_rule)) == 0) {
+      *matched_rule_length = (size_t) strlen(known_rule);
+      return true;
+    }
+  }
+  return false;
```

Comment:
Should we just set `matched_rule_length` to zero here as well as just a safety? Will enforce that it is always initialized to some value after the call.